Makefile

```diff
@@ -406,9 +406,8 @@ proto-all: proto-format proto-lint proto-gen
 
 proto-gen:
 	@echo "Generating Protobuf files"
-#	@if docker ps -a --format '{{.Names}}' | grep -Eq "^${containerProtoGen}$$"; then docker start -a $(containerProtoGen); else docker run --name $(containerProtoGen) -v $(CURDIR):/workspace --workdir /workspace $(protoImageName) \
-#		sh ./scripts/protocgen.sh; fi
```

Comment:
why this was commented?

Reply:
to generate form with pulsar. the docker image needed to be updated.